app/main/errors.py

```diff
@@ -17,11 +17,19 @@ def page_not_found(e):
 
 
 @main.app_errorhandler(500)
-def page_not_found(e):
+def exception(e):
     return _render_error_page(500)
 
 
-def _render_error_page(status_code):
+@main.app_errorhandler(503)
+def service_unavailable(e):
+    if e.response:
```

Comment:
I don't think we need this check - `_render_error_page` will handle the case if `error_message=None` anyway. So looks like this method should be able to just `return _render_error_page(503, e.response)` without the conditional.